ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-    <filter root="/apps/wknd"/>
+    <filter root="/apps/wknd/clientlibs"/>
```

Comment:
What's the violation of using `/apps/wknd` ? 

`/apps/wknd` only has these three sub-paths, so what the advantage of enumerating each?

Reply:
it explicitly avoid a potential overlap with the `ui.config` module which contains the path `/apps/wknd/osgiconfig`